assets/src/public.js

```diff
@@ -521,13 +521,16 @@ jQuery(function($){
 	
 					//types could be undefined
 					if (!obj.types) obj.types = [];
-					let isMeetingTempClosedAndOnline = (obj.types.indexOf('ONL') !== -1) && (obj.types.indexOf('TC') !== -1)
-					console.log(obj.name, isMeetingTempClosedAndOnline)
 					//add type 'flags'
 					if (typeof tsml.flags == 'object') {
+						// True if the meeting is temporarily closed, but online option available
+						var meetingIsOnlineAndTC = (obj.types.indexOf('TC') !== -1) && (obj.types.indexOf('ONL') !== -1)
 						for (var i = 0; i < tsml.flags.length; i++) {
-
-							if ((obj.types.indexOf(tsml.flags[i]) !== -1) && (isMeetingTempClosedAndOnline && tsml.flags[i] !== 'TC')) {
+							flagIsTempClosed = tsml.flags[i] === 'TC'
```

Comment:
prob need a `var` here for consistency

Reply:
I agree! Not sure why that didn't error?